perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -435,9 +435,18 @@ protected void doOptions(HttpServletRequest req, HttpServletResponse resp) throw
 
 	private void mirrorOriginHeader(HttpServletRequest req, HttpServletResponse resp) {
```

Comment:
The method no longer mirrors the header, so it should be renamed.

Reply:
Changed to ```checkOriginHeader()```.